DesktopEdge/MainWindow.xaml

```diff
@@ -223,7 +231,7 @@
         <local:IdentityDetails x:Name="IdentityMenu" Visibility="Collapsed" OnMFAToggled="MFAToggled" Authenticate="ShowAuthenticate" Recovery="ShowRecovery" OnLoading="DoLoading" OnShowMFA="ShowAuthenticate"></local:IdentityDetails>
 
         <!-- Modal Background -->
-        <Rectangle x:Name="ModalBg" MouseDown="Window_MouseDown" Margin="10,10,10,10" Fill="Black" RadiusY="10" RadiusX="10" Opacity="0" Visibility="Collapsed"></Rectangle>
+        <Rectangle x:Name="ModalBg" Margin="10,10,10,10" Fill="Black" RadiusY="10" RadiusX="10" Opacity="0" Visibility="Collapsed"></Rectangle>

```

Comment:
Removes unnecessary capability to attach/detach the main window from this modal.  Allowing this function here is a departure from the normal area where it is done (title area of each view).  Modals and pop-ups are temporary views that don't need such functionality.